packages/@react-spectrum/alert/package.json

```diff
@@ -4,8 +4,13 @@
   "description": "Spectrum UI components in React",
   "license": "Apache-2.0",
   "private": true,
-  "main": "dist/Alert.js",
-  "module": "dist/Alert.module.js",
+  "main": "dist/main.js",
```

Comment:
just double checking (i looked at the repo too) I think this is correct, but calling it out because the previous value was not like our other packages